### PR TITLE
Hello Orks line has been added to our code

### PR DESCRIPTION
**Pull request by daniiar2204**
Please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # stado-orkov
 Tut pasutsya vse orki
+Public static void main (Strings args[]){
+	System.out.println("Hello Orks");
```
Comment (msk77) [inline README.md]: // this is not a common phrase to say welcome to Orks

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # stado-orkov
```
Comment (msk77) [inline README.md:1]: V tselom very good

**Review by msk77 — CHANGES_REQUESTED** [review]
Сделай исправления и отправь на ревью заново

**Comment by daniiar2204** [comment]
Required changes have been added please review

**Review by msk77 — APPROVED** [review]
LGTM

**Pull request merged**